### ISSUE TITLE
Ace-Step-1.5 Model is not set to eval() mode, causing degradation?

### ISSUE THREAD

**Issue by scottmudge** [Comfy-Org/ComfyUI#12399]
### Custom Node Testing

- [x] I have tried disabling custom nodes and the issue persists (see [how to disable custom nodes](https://docs.comfy.org/troubleshooting/custom-node-issues#step-1%3A-test-with-all-custom-nodes-disabled) if you need help)

### Expected Behavior

The Ace-Step-1.5 model being loaded by the native nodes should be set to `.eval()` mode, but it is not.

### Actual Behavior

I noticed degradation in Ace-Step-1.5 after consecutive generations. Completely reloading the model appeared to correct the issue. This led me to look into this a bit deeper, and it looks like the Ace-Step-1.5 DiT model is not set to `.eval()` mode by the time it hits the sampler. Nor is it set to `.eval()` after the sampler, or after consecutive runs.

I suspect this is causing degradation in performance over time.

### Steps to Reproduce

Just using the [official workflows](https://comfyui.org/en/ace-step-15-is-now-available-in-comfyui) with the latest repository tip.

### Debug Logs

There is no 'official' output, I just created a simple node to inspect to see if `.training` was set in the model modules, and found these were not set to `.eval()` mode:

```
"model.model.ACEStep15",
"model.model.model_sampling.ModelSampling"
```

I create another simple passthrough node to force `.eval()` on the model, and the `.training` attributes disappeared.

**Comment by comfyanonymous** [comment]
Core comfyui sets every diffusion model to eval mode, it also runs everything in inference mode. Are you sure you didn't mess something up with custom nodes?

**Comment by scottmudge** [comment]
Yeah I saw that here:

```
if not unet_config.get("disable_unet_model_creation", False):
            if model_config.custom_operations is None:
                fp8 = model_config.optimizations.get("fp8", False)
                operations = comfy.ops.pick_operations(unet_config.get("dtype", None), self.manual_cast_dtype, fp8_optimizations=fp8, model_config=model_config)
            else:
                operations = model_config.custom_operations
            self.diffusion_model = unet_model(**unet_config, device=device, operations=operations)
            self.diffusion_model.eval()
            if comfy.model_management.force_channels_last():
                self.diffusion_model.to(memory_format=torch.channels_last)
                logging.debug("using channels last mode for diffusion model")
            logging.info("model weight dtype {}, manual cast: {}".format(self.get_dtype(), self.manual_cast_dtype))
            comfy.model_management.archive_model_dtypes(self.diffusion_model)
```

I used a pretty simple node:

```
class ForceEvalMode:
    @classmethod
    def INPUT_TYPES(cls):
        return {
            "required": {
                "model": ("*", {}),
            }
        }
    
    RETURN_TYPES = ("*",)
    RETURN_NAMES = ("model",)
    FUNCTION = "force_eval"
    
    def force_eval(self, model):
        # Try to find the actual PyTorch module and set it to eval
        if isinstance(model, torch.nn.Module):
            if model.training:
                model.eval()
                print("[ForceEvalMode] Set model to eval mode")
        elif hasattr(model, 'model'):
            if isinstance(model.model, torch.nn.Module):
                print("[ForceEvalMode] model.model exists")
                if model.model.training:
                    model.model.eval()
                    print("[ForceEvalMode] Set model.model to eval mode")
            if hasattr(model.model, 'diffusion_model') and isinstance(model.model.diffusion_model, torch.nn.Module):
                print("[ForceEvalMode] model.model.diffusion_model exists")
                if model.model.diffusion_model.training:
                    model.model.diffusion_model.eval()
                    print("[ForceEvalMode] Set model.model.diffusion_model to eval mode")    
            if hasattr(model.model, 'model_sampling') and isinstance(model.model.model_sampling, torch.nn.Module):
                print("[ForceEvalMode] model.model.model_sampling exists")
                if model.model.model_sampling.training:
                    model.model.model_sampling.eval()
                    print("[ForceEvalMode] Set model.model.model_sampling to eval mode")
                    
        if hasattr(model, 'diffusion_model') and isinstance(model.diffusion_model, torch.nn.Module):
            print("[ForceEvalMode] model.diffusion_model exists")
            if model.diffusion_model.training:
                model.diffusion_model.eval()
                print("[ForceEvalMode] Set model.diffusion_model to eval mode")
        
        return (model,)
    
```

Which outputs:

```
[ForceEvalMode] model.model exists
[ForceEvalMode] Set model.model to eval mode
[ForceEvalMode] model.model.diffusion_model exists
[ForceEvalMode] model.model.model_sampling exists
```
So `model.model` is not in `eval()` mode, but the others are. So maybe that doesn't matter since `model.model.diffusion_model` is set to `.eval()` mode. I'll close this for now, I definitely notice a strange degradation after a few generations, but maybe it's something else.

I already hashed everything else in the model to make sure nothing is changing, but everything seems consistent. Not sure what it is. I suppose "degradation" is a bit subjective, so it's difficult to really say for sure.
